apps/libsignpost/at_command.c

```diff
@@ -35,16 +35,17 @@ static int check_custom_buffer(uint8_t* buf, int len, const char* rstring) {
 }
 
 int at_send(int console_num, const char* cmd) {
-    console_write(console_num, (uint8_t*) cmd, strlen(cmd));
-    return AT_SUCCESS;
+    int ret = console_write(console_num, (uint8_t*) cmd, strlen(cmd));
+    if(ret < 0) return AT_ERROR;
+    else return ret;
```

Comment:
Is there a reason to change the error code here? Feels like you could just return the console error directly. This masks _why_ the console had an error and makes it harder for the caller of `at_send` to debug?